Typescript_Assignment/example.ts

```diff
@@ -0,0 +1,679 @@
+// Assignment
+// Complete the following MyArray class in TypeScript that can handle string and number only to get
+// the expected results
+// Barebone class definition
+// class MyArray {
+//  add(value) {
+//  this.collection.push(value)
+//  }
+//  remove(value) {}
+//  getValues() {
+//  return this.collection
+//  }
+// }
+// Expected results
+// const stringAry = new MyArray()
```

Comment:
This is not as per the type script syntax

Reply:
I have pasted every Question from question bank and below that I have written the Ans. Same goes to whole assignment.